spec/models/miq_expression_spec.rb

```diff
@@ -1631,4 +1631,30 @@
       end
     end
   end
+
+  describe "field_in_sql?" do
+    it "should returns false for model.virtualfield" do
```

Comment:
Very minor, but it may be better to avoid using "should" in a test description: http://betterspecs.org/#should